src/main/java/jenkins/metrics/api/Metrics.java

```diff
@@ -409,6 +433,10 @@ public Timer getHealthCheckDuration() {
         }
 
         public Map<String, HealthCheck.Result> getHealthCheckResults() {
```

Comment:
this method was introduced in the previous version (just a few weeks ago though), would break api, is that ok?